apps/server/src/modules/h5p-editor/controller/h5p-editor.controller.ts

```diff
@@ -172,16 +172,14 @@ export class H5PEditorController {
 	@Get('/edit')
 	@ApiResponse({ status: 200, type: H5PEditorModelResponse })
 	async getNewH5PEditor(@CurrentUser() currentUser: ICurrentUser) {
-		// TODO: Get user language
-		const editorModel = await this.h5pEditorUc.getEmptyH5pEditor(currentUser, 'de');
+		const editorModel = await this.h5pEditorUc.getEmptyH5pEditor(currentUser);
```

Comment:
```suggestion
		const editorModel = await this.h5pEditorUc.getEmptyH5pEditor(currentUser);
```
Why don't you use language, not as query param of client? Let's talk about it

Reply:
@SevenWaysDP For the endpoints 'getNewH5PEditor' and 'getH5PEditor' the language is passed as parameter.
This is not possible for the 'getAjax' endpoint, because the endpoint is only called internally by Lumi.

Reply:
@SteKrause sorry but your AjaxGetQueryParams has language as prop. Can you forward to lumi in client.